<launch>
    <node pkg="disturbance_estimation" type="disturbance_est_imu" name="disturbance_est_imu_node" output="screen">
        <param name="p1" type="double" value="-6.939e-7" />
        <param name="p2" type="double" value="3.127e-4" />
        <param name="p3" type="double" value="-0.2451" />
        <param name="p4" type="double" value="1.1122" />
        <param name="m" type="double" value="7.844" />
    </node>
</launch>
